lib/components/admin/field-trip-details.js

```diff
@@ -99,7 +99,7 @@ class FieldTripDetails extends Component {
     const { request, sessionId } = this.props
     const cancelled = request.status === 'cancelled'
     const printFieldTripLink = `/#/printFieldTrip/?requestId=${request.id}&sessionId=${sessionId}`
-    const newWindowIconWithStyles = (
+    const StyledNewWindowIcon = () => (
       <NewWindowIconA11y size={10} style={{ margin: '-3px 0 0 4px' }} />
     )
```

Comment:
Wait this still isn't a component! I'm actually confused how this works at all -- the definition looks ok but the new component should be used like `<StyledNewWindowIcon />`